index.html

```diff
@@ -460,31 +459,28 @@ <h2>
         </p>
       </div>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn>within scope</dfn> of
-        <a>navigation scope</a> <var>scope</var> if the following algorithm
-        returns <code>true</code>:
+        A [=URL=] |target:URL| is said to be <dfn data-dfn-for="URL"
+        data-export="">within scope</dfn> of [=URL=] |scope:URL| if the
+        following algorithm returns `true`:
       </p>
-      <ol>
-        <li>Let <var>scopePath</var> be the [=string/concatenation=] of
-        <var>scopes</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-        U+002F (/).
+      <ol class="algorithm">
+        <li>If |target| and |scope| are not [=same origin=], return `false`.
         </li>
-        <li>Let <var>targetPath</var> be the [=string/concatenation=] of <var>
-          target</var>'s <a data-cite="URL#concept-url-path">path</a>, using
-          U+002F (/).
+        <li>Let |scopePath:string| be the [=string/concatenation=] of
+        |scopes|'s [=URL/path=] using U+002F (/) as the separator.
         </li>
-        <li>If <var>target</var> is <a>same origin</a> as <var>scope</var> and
-        <var>targetPath</var> starts with <var>scopePath</var>, return
-        <code>true</code>.
+        <li>Let |targetPath:string| be the [=string/concatenation=] of
+        |target|'s [=URL/path=] using U+002F (/) as the separator.
         </li>
-        <li>Otherwise, return <code>false</code>.
+        <li>Return a [=boolean=] indicating whether |targetPath| starts with
+        |scopePath:string|.
         </li>
       </ol>
       <p>
-        A <a>URL</a> <var>target</var> is said to be <dfn data-lt=
-        "within-scope-manifest">within scope of a manifest</dfn>
-        <var>manifest</var> if <var>target</var> is <a>within scope</a> of the
-        navigation scope of <var>manifest</var>.
+        A [=URL=] |target:URL| is <dfn data-export="">within scope of a
+        manifest</dfn> if the |target| is [=URL/within scope=] of the
+        [=navigation scope=] (i.e., [=URL/within scope=] of the
```

Comment:
"of the navigation scope of _manifest_ (i.e., within scope of _manifest_'s `scope` member)."

Slight incongruity with the fact that "navigation scope" isn't defined as being "of a manifest", it just is a thing. But you have to say "of the manifest" here, otherwise we aren't defining "within scope of a manifest", just "within scope (of an implied context)".

Probably, we should change the definition of "navigation scope" to be "the navigation scope of a manifest is ...".

Reply:
Ok, so this will become:

```
<dfn data-dfn-for="manifest"
         data-lt="navigation scope"
         data-export="">navigation scope of a |manifest|</dfn>
```

And it can be used like this:

```
The [=manifest/navigation scope=] a |manifest| or [=processed manifest=]. 
```